Code/Editor/MainWindow.cpp

```diff
@@ -845,78 +852,183 @@ void MainWindow::InitActions()
         .SetShortcut(tr("Z"))
         .SetToolTip(tr("Center on Selection (Z)"))
         .Connect(&QAction::triggered, this, &MainWindow::OnGotoSelected);
+
+    auto goToViewBookmark = [](int index)
```

Comment:
nit: This could be marked const and I usually add a `Fn` postfix to these lambdas, so this would become...

```
const auto goToViewBookmarkFn = ....
```